docs/SlavaUkraini/examples/video-overlay/index.md

```diff
@@ -3,7 +3,68 @@ layout: tutorial_v2
 title: Leaflet on Mobile
```

Comment:
Side note: It seems this title has been wrong for quite some time, it should be updated.

Reply:
Yeah, I changed the name to _title: Overlay_, but not sure if it is good enough.

Reply:
Yeah, I changed it to the simple _title: Overlay_, but not sure if it is good enough to represent the tutorial.